databricks/koalas/generic.py

```diff
@@ -1515,9 +1515,10 @@ def _count_expr(col: spark.Column, spark_type: DataType) -> spark.Column:
 def _resolve_col(kdf, col_like):
     if isinstance(col_like, ks.Series):
         if kdf is not col_like._kdf:
-            raise ValueError(
-                "Cannot combine the series because it comes from a different dataframe. "
-                "In order to allow this operation, enable 'compute.ops_on_diff_frames' option.")
+            if not isinstance(kdf.columns, pd.MultiIndex):
```

Comment:
In fact, looks we're not currently supporting groupby with different dataframes when `compute.ops_on_diff_frames` is enabled. Seems like we should fix this first.